packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -124,7 +124,7 @@ class MediaQueryData {
       invertColors = window.accessibilityFeatures.invertColors,
       disableAnimations = window.accessibilityFeatures.disableAnimations,
       boldText = window.accessibilityFeatures.boldText,
-      highContrast = false,
+      highContrast = window.accessibilityFeatures.highContrast,
```

Comment:
```suggestion
      highContrast = false,
```

Hey @miquelbeltran, Can you revert this change? To make it less breaking, we should add the attribute before implementing it anywhere. That would break this up into three changes, a framework change that adds `highContrast` in `window_test.dart`, then the engine change should be unblocked, and then you can make another framework change that sets it here in `media_query.dart`.

I believe that should untangle this for you. Cc'ing @dnfield who may also have some thoughts about how to land this change.

Reply:
We should definitely update https://github.com/flutter/engine/blob/157772a31597e16ec8fc19be1b35983c0294a7ee/lib/ui/window.dart#L1208 first.

Reply:
@Piinks yes, I can separate this PR into two:

1. One that updates `window_test.dart`
2. One that updates `media_query.dart`

However as @dnfield points, it will still be a breaking change because the `window_test.dart` depends on the `flutter/engine` change, and as well, the `flutter/engine` change depends on `window_test.dart`. We have a cyclic dependency here.

So, it is still worth it to separate the PR into two if we still have the cyclic dependency from the test part?

Reply:
The test window implements noSuchMethod so that it won't break in cases like this

Reply:
alright! I applied the suggestion, and will open a new PR for this change